packages/cloud_firestore/cloud_firestore_web/pubspec.yaml

```
@@ -14,16 +14,17 @@ dependencies:
     sdk: flutter
   http_parser: ^3.1.3
   meta: ^1.1.7
-  firebase_core: ^0.7.0
-  firebase_core_web: ^0.2.1+3
+  firebase_core: ^0.7.0-nullsafety+0
```

```suggestion
  firebase_core: ^0.7.0-nullsafety.0
```

There's 29 of these in total so won't spam you with all of them 😅

---

haha sorry!

---

No worries 😄